compiler-plugin/src/test/kotlin/arrow/meta/plugins/comprehensions/ComprehensionsTest.kt

```diff
@@ -49,7 +49,7 @@ class ComprehensionsTest {
         listOf(addCompilerPlugins(compilerPlugin))
       },
       code = {
-        codeSnippet.source
```

Comment:
I'd keep it because of: avoiding the use of primitives types, better types and it's planned to have several sources.

Reply:
May you reiterate on what you mean with multiple sources?

Reply:
I mean about different snippets.

Reply:
that should be isomorphic to `List<Source>`. What am indicating here is that your not doing anything to your DataType which is domain-specific other than renaming `String` to `Source`